src/main/java/seedu/address/logic/commands/CommandWord.java

```diff
@@ -21,6 +21,9 @@
     LIST("list"),
     VIEW("view"),
     SORT("sort"),
+    REMIND("remind"),
+    REMINDER("listr"),
+    SHOW_REMINDER("showr"),
```

Comment:
What I meant was to split it into command _word_ and command _types_. Not to modify the command _word_. So right now contact commands have the _type_ "-c", event commands "-e", and tag commands "-t". You can add your own and make a reminder oriented set of commands. So reminder commands can have the command _type_ "-r". This means u can reuse command _words_ like "add", combining them with the "-r" command _type_ and it becomes "add -r"